components/libwebsockets/idf_component.yml

```
@@ -0,0 +1,5 @@
+version: "0.1.0"
+url: https://github.com/espressif/esp-protocols/tree/master/components/libwebsockets
+description: The component provides a simple ESP32 port of libwebsockets server and client.
```

```suggestion
description: The component provides a simple ESP-IDF port of libwebsockets server and client.
```
or maybe remove also "server and client" (as it's just a client for now?)

---

done, thanks